templates/tease-taxonomy-post.twig

```diff
@@ -1,5 +1,6 @@
 <li id="result-row-{{ post.ID }}" class="media search-result-list-item">
-	<img class="d-flex search-result-item-image" src="{{ post.thumbnail.src('thumbnail') ?? dummy_thumbnail }}"
+	<img class="d-flex search-result-item-image lazyload"
+		data-src="{{ post.thumbnail.src('thumbnail') ?? dummy_thumbnail }}"
```

Comment:
I think you also need to modify the failing [unit test](https://github.com/greenpeace/planet4-master-theme/blob/develop/tests/test-category-page.php#L41-L48), because it looks for an src tag to get the thumbnail image.

Reply:
Ahhh right, didn't see that, fixing now